ofrak_patch_maker/ofrak_patch_maker/toolchain/utils.py

```diff
@@ -48,10 +51,18 @@ def get_repository_config(section: str, key: Optional[str] = None):
         try:
             config.read(conf)
             if key:
-                ret = config.get(section, key)
+                raw_value = config.get(section, key)
+
+                if "*" in raw_value:
+                    matches = glob.glob(raw_value)
+                    if matches:
+                        return matches[0]
+                    else:
+                        raise NotFoundError(f"No file matches wildcard {raw_value}")
+                else:
+                    return raw_value
```

Comment:
I don't feel very positively about this change to use globbing. I personally feel that it makes the config file harder to read and understand. It also seems kind of error-prone for anyone not running OFRAK inside of Docker, who may have their own toolchains installed on their system.

@Jepson2k @alchzh what alternatives were considered here instead of globbing?